packages/shared/src/components/plus/PlusDesktop.tsx

```diff
@@ -28,12 +31,32 @@ export const PlusDesktop = (): ReactElement => {
       return;
     }
 
+    if (giftToUser) {
+      if (!giftOneYear) {
+        return;
+      }
+
+      if (!selectedOption) {
+        const { value } = giftOneYear;
+        setSelectedOption(value);
+        openCheckout({ priceId: value });
+      }
+
+      return;
+    }
```

Comment:
Do we already have this logic in the mobile component?

Reply:
I checked the mobile component, seems to be not used. Maybe because we open checkout there after submit button.